hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/validators/CustomFeesValidatorTest.java

```diff
@@ -176,7 +175,7 @@ void royaltyFeeFailsFungibleDenomOnFeeScheduleUpdate() {
                         writableTokenStore,
                         feeWithRoyalty))
                 .isInstanceOf(HandleException.class)
-                .has(responseCode(CUSTOM_FEE_DENOMINATION_MUST_BE_FUNGIBLE_COMMON));
+                .has(responseCode(INVALID_TOKEN_ID_IN_CUSTOM_FEES));
```

Comment:
@Neeharika-Sompalli is this a correct change?

Reply:
Yeah we have to be very careful about correcting the tests. @Neeharika-Sompalli any thoughts?

Reply:
Agree with @mhess-swl . We should not change response codes on mono-service tests. We should comply with mono-service. I believe the change in `CustomFeesValidator` also should be reverted.